apps/ui/components/search/ReservationUnitCard.tsx

```
@@ -79,6 +80,19 @@ export function ReservationUnitCard({
       }),
     });
   }
+  if (reservationUnit.currentAccessType) {
+    infos.push({
+      icon: (
+        <IconLock
+          aria-label={t("reservationUnit:accessType")}
+          size={IconSize.Small}
+        />
+      ),
+      value: t(
+        `reservationUnit:accessTypes.${reservationUnit.currentAccessType}`
```

can this be null?

---

Shouldn't be possible to my knowledge, as all reservation units have an `accessType` (which `currentAccessType` defaults to, if it's not specified separately)

---

about to change

---

shall be null until the information has been filled in on the reservation units (esp. production) since even disregarding keyless access some spaces are currently opened by staff or physical key needs to be procured beforehand, so the backend defaulting to direct access would be misleading